joomla/joomla-cms
# [4.0] Table Options - Inc. com_menu & small screen devices

Pull Request for Issue  #17110 .

### Summary of Changes
Adds order and list dropdowns to 'Table Options' in com_menu. Make 'Table Options' available on small screen devices.

### Testing Instructions
Ensure 'Table Options' displays correctly on com_menus (all list views). Ensure visible on all screen sizes.

![com_menus2](https://user-images.githubusercontent.com/2803503/28180268-853afc6e-67fc-11e7-92da-f75c3a384143.png)


### Documentation Changes Required
Yes
